.github/workflows/beam_PreCommit_Java.yml

```diff
@@ -177,6 +177,9 @@ jobs:
           github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
       - name: Setup environment
         uses: ./.github/actions/setup-environment-action
+        with:
+          python-version: 3.9
```

Comment:
we should not need to pin a python version here. We should update ARC base image if it is stale

cc: @Amar3tto

Reply:
`python-version: default` should also work (3.9).
As for upgrading the base ARC image: this is potentially a breaking change for many workflows and should be thoroughly tested if we decide to update it.
I would suggest using `python-version: default` because as I know the basic idea is that if we need to install python, we should always explicitly declare it.

Reply:
Sounds good. Thanks.